src/service/world-api-adapter.js

```diff
@@ -70,20 +106,27 @@ module.exports = function (config) {
     var publicAPI = {
 
         /**
-        * Create a new World
+        * Create a new World.
         *
-        * ** Example **
-        *   var gm = new F.service.World({ account: 'account', project: 'project' });
-        *   gm.create({ model: 'model.py', group: 'group-name' });
+        *  **Example**
         *
-        * ** Parameters **
-        * @param {object} `params` Parameters to create the world
-        * @param {string} `params.model` The model file to use to create runs in this world
-        * @param {string} `params.group` the group _name_ to create this world under
-        * @param {string} `params.roles` (Optional) The list of roles for this world
-        * @param {string} `params.opionalRoles` (Optional) The list of optional roles for this world
-        * @param {string} `params.minUsers` (Optional) The minimum number of users for the world
-        * @param {object} `options` Options object to override global options
+        *      var gm = new F.service.World({ 
```

Comment:
nitpick - if we're calling it a *World* service we shouldn't use `gm` in the examples :)